tango_simlib/utilities/helper_module.py

```diff
@@ -179,7 +183,7 @@ def json_loads_byteified(json_text):
 
 def _byteify(data, ignore_dicts=False):
     """If this is a unicode string, return its string representation."""
-    if isinstance(data, unicode):
+    if isinstance(data, str):
```

Comment:
Good point. Python 3 `str` is Unicode.
Similar "byteify" code was recently removed from the SKA LMC base classes repo because it now only has to support Python 3.  Looks like PyTango might take care of this in Python 3.  So, you can try only doing the byteify work in Python 2.